apps/contrib/formset.py

```diff
@@ -0,0 +1,22 @@
+from django import forms
+
+
+class DynamicModelFormSet(forms.BaseModelFormSet):
+    def get_default_prefix(cls):
+        return 'test'
+
+    @property
+    def media(self):
+        additional_js = forms.Media(js={
+            'formset.js'
+        })
+        return super().media + additional_js
+
+
+def dynamic_modelformset_factory(*args, **kwargs):
+    kwargs.update({
+        'formset': DynamicModelFormSet
+    })
+    modelformset = forms.modelformset_factory(*args, **kwargs)
+
+    return modelformset
```

Comment:
If I see correctly the whole purpose of this file is to inject `formst.js` and `get_default_prefix`, right? Is there no simpler way to do that?